src/components/molecules/BottomNavBar.tsx

```diff
@@ -1,66 +1,74 @@
 import { House, ChatsCircle, User } from '@phosphor-icons/react';
 import * as S from '../../styles/molecules/BottomNavBar';
-import { useState } from 'react';
-import { useNavigate } from 'react-router-dom';
+import { useState, useCallback } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import React from 'react';
 
 // 하단 네비게이션 바
 const BottomNavBar = () => {
-  const [active, setActive] = useState(0);
+  const [active, setActive] = useState('');
   const navigate = useNavigate();
+  const location = useLocation();
+
+  const useQuery = () => {
+    return location.pathname;
+  };
+
+  console.log(useQuery());
 
   // 클릭 시 해당 페이지로 이동
-  const handleActive = (index: number) => {
-    setActive(index);
+  const handleActive = useCallback((page: string) => {
+    setActive(page);
 
-    // switch (index) {
-    //   case 0:
-    //     navigate('/')
-    //     break;
-    //   case 1: // chat
-    //     navigate('/chat')
-    //     break;
-    //   case 2: // profile
-    //     console.log('/profile');
-    //     break;
-    //   default:
-    //     break;
-    // }
-  };
+    switch (page) {
+      case '/':
+        navigate('/');
+        break;
+      case '/chatlist': // chat
+        navigate('/chatlist');
+        break;
+      case '/profile': // profile
+        navigate('/profile');
+        break;
+      default:
+        break;
```

Comment:
```suggestion
    navigate(page);
```

그냥 이렇게 처리하면 안되는 부분일까요?
switch-case 문을 작성한게 의미가 없어보여서요.
다른 고려사항이 있다면 알려주세요~

Reply:
코드 작성할 때는 나누기 편하니까 switch 문을 썼었던건데,
확인해보니 정말 switch문을 쓰는 의미가 없었네요
알려주신 방법으로 수정하였습니다!